spec/unit/recipes/_configure_indexes_spec.rb

```diff
@@ -134,6 +134,91 @@
         }
       }
 
+      expect(subject).to create_splunk_template('system/indexes.conf').with(expected_attributes)
+    end
+  end
+  # When _noGenerateTstatsHomePath => true in specific indexes, then writes the indexes.conf file without tstats for only for that indexes
```

Comment:
Instead of writing this as a comment, can you update the `context` and `it` accordingly.

```
context 'when _noGenerateTstatsHomePath is set to true for specific indexes' do
    it 'writes the indexes.conf file without tstatsHomePath only for those indexes' do
```

Same comment applies to the next test.

Reply:
removed the comment and updated in context and it.